package.json

```diff
@@ -1,8 +1,8 @@
 {
     "devDependencies": {
-        "@rocket.chat/apps-engine": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#master",
+        "@rocket.chat/apps-engine": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#multicode_setting_type",
```

Comment:
We have now merged `multicode_setting_type` in `master`, So we should be good with the master branch. Just make sure it has the latest commit in package-lock.json file

Reply:
@AlexanderKanakis @Shailesh351 looks like this one last change is required then we're good to go?

Reply:
Ready.

Reply:
@AlexanderKanakis Can you update the package-lock.json file as well

Reply:
Currently the hash is `41bdb50dd83c798796ff781fff40323b52590489 ` https://github.com/WideChat/Apps.Dialogflow/blob/300ef80e6016c687b2762c2986bb13034a62bc43/package-lock.json#L32

and latest on apps-engine is `7777d4b7702500a3c2ff28aaae9fa8963dba109f`

I think we haven't updated it on the DF App repo

Reply:
Updated apps-engine to `7777d4b7702500a3c2ff28aaae9fa8963dba109f`

Reply:
Thanks, @AlexanderKanakis! 🙌